src/plugins/vis_type_timelion/public/components/timelion_vis_component.tsx

```diff
@@ -38,15 +39,16 @@ import { Series, Sheet } from '../helpers/timelion_request_handler';
 import { tickFormatters } from '../helpers/tick_formatters';
 import { generateTicksProvider } from '../helpers/tick_generator';
 import { TimelionVisDependencies } from '../plugin';
-import { ExprVisAPIEvents } from '../../../visualizations/public';
+
+import './index.scss';
 
 interface CrosshairPlot extends jquery.flot.plot {
   setCrosshair: (pos: Position) => void;
   clearCrosshair: () => void;
 }
 
-interface PanelProps {
-  applyFilter: ExprVisAPIEvents['applyFilter'];
+interface TimelionVisComponentProps {
+  fireEvent(event: any): void;
```

Comment:
IInterpreterRenderHandlers ?

Reply:
There is no difference between types,
but from a side of make it more centralized this would make sense